src/assets/json/communities.json

```diff
@@ -258,7 +258,42 @@
     "organizers": [
       {
         "name": "Michael Hladky",
-        "twitter": "https://twitter.com/michael_hladky"
+        "twitter": "https://twitter.com/michael_hladky",
+        "webs": ["https://www.linkedin.com/in/michael-hladky-519340148/"]
+      },
+      {
+        "name": "Miroslav Jonas",
+        "twitter": "https://twitter.com/meeroslav/",
+        "webs": ["https://missing-manual.com/"]
+      }
+    ]
+  },
+  "Angular Austria": {
```

Comment:
4. is AngularAustria is a local-community which do events about Angular? Is this community related with Angular-Vienna ?